Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM centos:7
+LABEL maintainer="tdockendorf@osc.edu; johrstrom@osc.edu"
+
+# setup the ondemand repositories
+RUN yum -y install https://yum.osc.edu/ondemand/latest/ondemand-release-web-latest-1-6.noarch.rpm
+
+# install all the dependencies
+RUN yum install -y centos-release-scl && \
+    yum -y update && \
+    yum install -y \
+        file \
+        lsof \
+        sqlite-devel \
+        sudo \
+        httpd24 \
+        rh-ruby25 \
+        rh-nodejs10 \
```

Comment:
Can replace these 3 SCLs with the following:

* ondemand-apache
* ondemand-ruby
* ondemand-nodejs